sentry_sdk/integrations/celery.py

```diff
@@ -114,6 +114,16 @@ def sentry_build_tracer(name, task, *args, **kwargs):
         ignore_logger("celery.redirected")
 
 
+def _now_seconds_since_epoch():
+    # type: () -> float
+    # We can not use `time.perf_counter()` when dealing with the duration
+    # of a Celery tasks, because the start of a Celery tasks and
+    # the end are recorded in differenct processes.
+    # Start happens in the Celery Beat process,
+    # the end in a Celery Worker process.
```

Comment:
```suggestion
    # We cannot use `time.perf_counter()` when dealing with the duration
    # of a Celery task, because the start of a Celery task and
    # the end are recorded in different processes.
    # Start happens in the Celery Beat process,
    # the end in a Celery Worker process.
```